src/Routes/SessionRoute/SessionRoute.js

```diff
@@ -273,6 +297,7 @@ componentDidMount() {
         </div>
         <div>
           <h2>Learning Target: </h2>
+          {completedText}
```

Comment:
With the most recent PR, if there is no active session goal the learning target, students,etc. won't how - just a message telling users to go set a learning target on their dashboard.